src/BlackSlope.Api.Common/Extensions/BlackSlopeBuilderExtensions.cs

```diff
@@ -11,7 +11,7 @@ public static IApplicationBuilder UseSwagger(this IApplicationBuilder app, Swagg
         {
             app.UseSwagger(c =>
             {
-                c.SerializeAsV2 = true;
+                c.SerializeAsV2 = false; // NOTE: Will serialize in latest OpenAPI version
```

Comment:
I guess default is false, and this is redundant.

Reply:
Yeah, I can scrub that. Wasn't sure if leaving it around to be descriptive was worth it or not.

Reply:
Patched. @pmotameni